r/R/dataset.R

```diff
@@ -131,8 +131,7 @@
 #' dir.create(tf)
 #' on.exit(unlink(tf))
 #'
-#' data <- dplyr::group_by(mtcars, cyl)
```

Comment:
The complete removal of `data` here breaks the example further down `write_dataset(data, tf2, format = "ipc")` - please can you update that too?

Reply:
Thanks for pointing that out. I've updated that example and tested locally on my computer to make sure it works now.